src/index.ts

```diff
@@ -1191,28 +1192,28 @@ export class KycDao extends ApiBase {
 
                 const updatedChainId = await this.evmProvider.getChainId();
                 if (updatedChainId !== newChainId) {
-                  throw new Error();
+                  throw new InternalError('ChainId expected to be changed.');
```

Comment:
This error message will be ignored in the catch, that's why I didn't bother to write a message for it :D